tests/components/shelly/snapshots/test_devices.ambr

```diff
@@ -571,38 +571,41 @@
     'name': None,
     'options': dict({
       'sensor': dict({
-        'suggested_display_precision': 0,
+        'suggested_display_precision': 2,
+      }),
+      'sensor.private': dict({
+        'suggested_unit_of_measurement': <UnitOfEnergy.KILO_WATT_HOUR: 'kWh'>,
       }),
     }),
-    'original_device_class': <SensorDeviceClass.FREQUENCY: 'frequency'>,
+    'original_device_class': <SensorDeviceClass.ENERGY: 'energy'>,
```

Comment:
I think the snapshot was edited manually and that's why now after using `--snapshot-update` the order of entries has changed.